torch/onnx/symbolic_opset9.py

```diff
@@ -837,7 +837,8 @@ def softmax(g, input, dim, dtype=None):
 @parse_args("v", "t", "v")
 def softplus(g, self, beta, threshold):
     if beta != 1:
```

Comment:
nit: we don't need `if beta != 1` condition after this changed right?

Reply:
Having this check will skip some nodes that are not needed